scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/ufqfpn.yaml

```diff
@@ -0,0 +1,58 @@
+UFQFPN-32-1EP_5x5mm_P0.5mm_EP3.5x3.5mm:
+  device_type: 'UFQFPN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.st.com/resource/en/datasheet/stm32g071k8.pdf'
+  ipc_class: 'ufqfpn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  # custom_name_format:
+  body_size_x:
+    nominal: 5
+  body_size_y:
+    nominal: 5
+  overall_height:
+    minimum: 0.5
+    nominal: 0.55
+    maximum: 0.6
+
+  lead_width:
+    minimum: 0.18
+    nominal: 0.23
+    maximum: 0.28
+  lead_len:
+    minimum: 0.3
+    nominal: 0.4
+    maximum: 0.5
+
+  EP_size_x:
+    minimum: 3.4
+    nominal: 3.5
+    maximum: 3.6
+  EP_size_y:
+    minimum: 3.4
+    nominal: 3.5
+    maximum: 3.6
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [2, 2]
+
+  thermal_vias:
+    count: [3, 3]
+    drill: 0.2
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    # EP_num_paste_pads: [2, 2]
+    #paste_between_vias: 1
+    #paste_rings_outside: 1
+    EP_paste_coverage: 0.75
+    #grid: [1, 1]
+    # bottom_pad_size:
+    paste_avoid_via: False
+
+  pitch: 0.50
+  num_pins_x: 8
+  num_pins_y: 8
+  chamfer_edge_pins: 0.09
+  #pin_count_grid:
+  #pad_length_addition: 0.5
+  #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
+  #include_suffix_in_3dpath: 'False'
```

Comment:
You can remove `chamfer_edge_pins`. The recommended layout only has this on Pin1.

Please also remove the unused lines here and above.